mesonbuild/dependencies/misc.py

```diff
@@ -474,6 +474,24 @@ def __init__(self, name: str, env: 'Environment', kwargs: T.Dict[str, T.Any]):
                     self.link_args.extend(sublib)
 
 
+class ObjFWDependencyConfigTool(ConfigToolDependency):
+
+    tools = ['objfw-config']
+    tool_name = 'objfw-config'
+
+    def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.Any]):
+        super().__init__(name, environment, kwargs)
```

Comment:
You need a `self.feature_since = ('1.5.0', '')` here so that Meson will warn if someone tries to use this on an old Meson

Reply:
How would that warn when someone is using an old meson that doesn't contain this check?

Reply:
It warns the other way, If you add this, but you've set your supported Meson version to say, 1.3, it will warn you that trying to find objfw will fail. ie:

```meson
project('test', 'objc', meson_version : '>=1.3')

dep = dependency('objfw')
```

Then this will warn you that you've created a configuration that will fail

Reply:
Ah, I see. In any case, I already added it.